newswires/client/src/urlState.ts

```diff
@@ -45,14 +47,20 @@ export const configToUrl = (config: Config): string => {
 };
 
 export const paramsToQuerystring = (config: Query): string => {
-	const params = Object.fromEntries(
-		Object.entries(config).reduce<Array<[string, string]>>((acc, [k, v]) => {
+	const params = Object.entries(config).reduce<Array<[string, string]>>(
```

Comment:
turning the `Array<[string, string]>>` into an object before creating the `new URLSearchParams` was causing us to remove repeated keys from the params, e.g.

`Object.entries([[a, 1], [b, 2], [a, 3]])` becomes `{b: 2, a: 3}`, rather than `{a: 1, b: 2, c: 3}`.

I can't work out though why this wasn't being caught by the new test case -- `'converts config with many suppliers to querystring'` in this branch...

Reply:
turns out it was being caught by the test case :D - I just hadn't run it locally, and the current CI definition doesn't run the client tests (probably due to there not being any before...)
fixed in latest commit